meta-lmp-base/recipes-support/fio-diag/fio-diag_0.1.bb

```diff
@@ -0,0 +1,19 @@
+SUMMARY = "foundries diagnostic tool"
+SECTION = "devel"
+LICENSE = "BSD-2-Clause"
```

Comment:
The reported license in the source code of the script is `BSD-3-Clause`

https://github.com/foundriesio/lmp-tools/blob/master/device-scripts/fio-diag.sh#L5